doc/source/whatsnew/v0.18.0.txt

```diff
@@ -248,6 +248,35 @@ Subtraction by ``Timedelta`` in a ``Series`` by a ``Timestamp`` works (:issue:`1
    ser
    pd.Timestamp('2012-01-01') - ser
 
+
+Signature change for Series.rank and DataFrame.rank and their GroupBy equivalents
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The **new** signature is ``.rank(method='average', axis=0, numeric_only=None, na_option='keep', ascending=True, pct=False)``.
+
```

Comment:
What about `Series.rank` then? If `axis` comes first, then it means the first parameter is the least useful...